tasks/copy_to_glacier/schemas/config.json

```diff
@@ -0,0 +1,48 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://github.com/nasa/cumulus-orca/blob/master/tasks/copy_to_files/schemas/config.json",
+  "title": "copy_to_files Lambda Config",
+  "description": "The config for the copy_to_files Lambda.",
+  "type": "object",
+  "required": ["collection"],
+  "properties": {
+    "collection": {
+      "description": "The collection from AWS.",
+      "type": "object",
+      "required": ["name", "version", "files", "url_path"],
```

Comment:
Looking at the code I do not believe any of these properties are needed or required. I believe all we really need is the `meta` property object and optionally the `excludeFileTypes` property list nested within the object.

Reply:
Hm. Looks like this got reworked at some point, and the properties weren't removed. Fixed.

Reply:
`meta` is optional as well.